tannerb9/Tandem-Trivia
# Fixes and more bugs

# Description
Fixed buttons so that answers and 'next/submit' button correctly toggle disabled depending on whether a question has or hasn't been answered. Fixed handleSubmit on TriviaForm component to display next question. 
Feat: modal displays upon round ending with user's score generated

## Type of change

# Testing Instructions
Play a round. 'Next' and 'Submit' button should be disabled if answer has yet to be chosen but should become enabled after answer chosen. 
Hitting 'Submit' after last question opens modal displaying your score for that round.
Bug: Hitting 'Playing Again?' button on modal close modal but doesn't start a new round.
